changes/fabo_changelog-tool

```diff
@@ -0,0 +1 @@
+[Added] Added a tool to handle changelog conflicts @faboweb
```

Comment:
shouldn't his go to `added/...`

Reply:
also should we rename the file ?

Reply:
I chose to go with `changes/{branch name}` so you are only responsible for the changes of your branch which shouldn't create conflicts. Going with `changes/added/....` we would create a lot more lookups and I think that would be more complicated to handle the folder structures.

Reply:
ohh ok, I wasn't aware of that naming standard. Yeah, I agree with your point, just let's make sure we document that on `CONTRIBUTING.md`